src/symbol.ts

```diff
@@ -1,14 +1,14 @@
-'use strict';
 /**
  * A class representation of the BSON Symbol type.
  */
-class BSONSymbol {
+export class BSONSymbol {
```

Comment:
how much of a breaking change is it to remove this type alltogether and depend on the `Symbol` type with a possible built-in polyfill? (similar to `Map`)

Reply:
The class's name is BSONSymbol and we export it as such, not sure how we can drop that. Also Symbols don't have a .value property, and they're not constructed i.e. `new Symbol` fails.